### PR TITLE
Now export only the app that is required

### PR DESCRIPTION
**Pull request by surajssd**
Added an option called as `--label-selector`, which acts as a filter using which only specific app can be exported, rather than exporting all the apps in the project.

Fixes issue #40

### CODE REVIEW

File: openshift2nulecule/openshift.py [hunk 1]
```diff
@@ -126,7 +128,9 @@ def export_project(self):
                              "route"]
 
             # output of this export is kind List
-            args = ["export", ",".join(resources), "-o", "json"]
+            args = ["export", ",".join(resources),
+                    "-l", self.label_selector,
```
Comment (kadel) [inline openshift2nulecule/openshift.py]: What if `--label-selector` is not specified from commandline?

Comment (surajssd) [inline openshift2nulecule/openshift.py]: Missed that, sorry will change that!

**Comment by kadel** [comment]
Can we also change `--label-selector` to just `--selector` so it is same as in `oc` and `kubectl`?

**Comment by surajssd** [comment]
Yes will take that into consideration.

**Comment by surajssd** [comment]
@kadel why it failed now?

**Comment by kadel** [comment]
this sometimes happens, I don't know why :cry:  
You can try to restart build.

**Pull request merged**